sklearn/tree/tests/test_tree.py

```diff
@@ -135,10 +135,12 @@
 X_multilabel, y_multilabel = datasets.make_multilabel_classification(
     random_state=0, n_samples=30, n_features=10)
 
+# NB: despite their names X_sparse_* are numpy arrays (and not sparse matrices)
```

Comment:
flake8 error(80 character?) or I'm wrong?

Reply:
I think it just fits within 79 characters :)